app/controllers/photos_controller.rb

```diff
@@ -32,6 +32,17 @@ def set_cover_photo
     json_response({success: true, message: "Set as cover photo successfully.", data: {photos: @photo}}, 200)
   end
 
+  def mark_as_checked
+    if params[:photo_id].present?
+      @photo = Photo.find_by(id: params[:photo_id])
+      if @photo.is_selected == false
+        @photo.update_attribute :is_selected, true
+      else
+        @photo.update_attribute :is_selected, false
+      end
+    end
+  end
+
```

Comment:
1) Please create fetch_photo and use this method in `set_cover_photo and mark_as_checked`, becuase this is using in two places `Photo.find(params[:id])`
2) Please comment route before action
3) No need to check for this condition `params[:photo_id].present?`